src/sagemaker/workflow/pipeline.py

```diff
@@ -106,7 +119,7 @@ def _create_args(self, role_arn: str, description: str):
 
         Args:
             role_arn (str): The role arn that is assumed by pipelines to create step artifacts.
-            pipeline_description (str): A description of the pipeline.
+            description (str): A description of the pipeline.
```

Comment:
nit - looks like this is just a doc change but ideally doc would be checked statically to avoid this